app/templates/ember_widgets/documentation.hbs

```diff
@@ -100,6 +100,11 @@
       <td>&#39;&#39;</td>
       <td><p>The path of the default option.</p></td>
     </tr>
+    <tr>
+      <td>showTooltip</td>
+      <td>true</td>
+      <td><p>Whether to show a tooltip when hovering over an option.</p></td>
```

Comment:
I would check with Patricia on this wording. Don't think she has access to ember widgets, but in the meantime, might make sense to just pull her aside for advice...